### ISSUE TITLE
OPTIONS in Routing.java

### ISSUE THREAD

**Issue by 2beers** [decebals/pippo#405]
You forgot to add OPTIONS to Routing.java as well https://github.com/decebals/pippo/blob/master/pippo-core/src/main/java/ro/pippo/core/route/Routing.java

As of 1.5.0 version

**Comment by decebals** [comment]
Closed with b1f5bcd.

**Comment by JackLeEmmerdeur** [comment]
Thanks for your hard work. It's, compared to others, delightful to work and tinker with the http-protocol.
I need to insert an global OPTIONS-Filter (similar to the log-example in your docs), before every POST/GET to negotiate the "Access-Control-Allow-Origin"/CORS-Preflight. I've implemented it for a single controller. The problem is as the OP posted, that the 1.5.0 version in the maven-repos, does not contain the OPTIONS default-interface-Method in Routing.java. Can I somehow patch it myself, or do I need to build the project?

**Comment by decebals** [comment]
@JackLeEmmerdeur Thanks for your nice words!
You can use the latest SNAPSHOT version. See http://www.pippo.ro/dev/maven.html for more details.

**Comment by JackLeEmmerdeur** [comment]
I cloned the repo before your answer. Looking at the amount of classes, the dryness of the code and how well the modularity and integration of the template/server/metrics/session/handler libraries was done, is exceptional. An herculean task for nearly only two people. I would've never, even with your knowledge in web-technology/java, given the thought of inventing another web-framework, a 2nd chance. Hope yours will gain more attraction.

Thanks for the quick answer. After looking at Routing.java, I already had the solution:
`super.addRoute(new Route("OPTIONS", "/.*", r -> {}));`

I added repositories-tag for sonatype, but <version>1.5.0</version> seems to be the most current for pippo there, too.
